docs/tutorials/modify_reports.md

```diff
@@ -1,10 +1,153 @@
+
+Here we demonstrate overriding the existing configuration of a report in a project frame that utilizes the Zambia 
+country model.
+
+The Zambia country model class exists in: ``emodpy_hiv/countries/zambia.py``. The following snippet from the Zambia
+country model indicates its default specification of the **InsetChart** report:
+
+```python 
+@classmethod
+def build_reports(cls, reporters: Reporters) -> Reporters:
+    reporters = super().build_reports(reporters=reporters)
+
+    # ...
+
+    reporters.add(InsetChart(reporters_object=reporters,
+                             has_ip=None,  # default
+                             has_interventions=None,      # default
+                             include_pregnancies=False,   # default
+                             include_coital_acts=True,    # default
+                             event_channels_list=["NewInfectionEvent"]))
+
+    # ...
+```
+
+If we wish to override this behavior in a project frame **without modifying the country model** we will need to override
+it in our frame. Reports are specified as part of the EMOD config file, so we edit the frame file:``config.py``. This 
+file typically contains (at minimum) the following function definition ready for us to use:
+
+```python
+def build_reports(reporters: Reporters):
+    reporters = country_model.build_reports(reporters)
+    return reporters
+```
+
+!!! Important
+    Each report can only have at most ONE configuration. Duplicative configurations will generate an error.
+
+We first remove the existing report that we do not want. Then we add a new definition of the report to replace it. For
+example, if we want to add information about pregnancies and potential mothers in the model, we copy/paste the 
+**InsetChart** definition from the Zambia country model and modify the appropriate parameter 
+(**include_pregnancies=True**) (labeled below with **NEW CODE**):
+
+```python
+def build_reports(reporters: Reporters):
+    from emodpy_hiv.reporters.reporters import InsetChart
+
+    reporters = country_model.build_reports(reporters)
+
+    # Remove default built-in InsetChart configuration
+    reporters.config_reporters = [reporter for reporter in reporters.config_reporters
+                                  if not isinstance(reporter, InsetChart)]
+
+    # Add new InsetChart configuration that records pregnancy information
+    reporters.add(InsetChart(reporters_object=reporters,
+                             has_ip=None,  # default
+                             has_interventions=None,  # default
+                             # NEW CODE
+                             include_pregnancies=True,  # NOT the default
+                             include_coital_acts=True,  # default
+                             event_channels_list=["NewInfectionEvent"]))
+
+    return reporters
+```
+
+Future executions of EMOD using this frame will now generate an ``InsetChart.json`` output file containing pregnancy
+related data.
+
+### Run EMOD
```

Comment:
I've seen both "run EMOD" and "plot insetchart.json" instructions in multiple places--these both might be good to make reusable snippets so we aren't duplicating them in multiple places.